modules/RTC/TraceablePeerConnection.js

```diff
@@ -769,45 +781,35 @@ TraceablePeerConnection.prototype.getRemoteTracks = function(endpointId, mediaTy
 };
 
 /**
- * Parses the remote description and returns the sdp lines of the sources associated with a remote participant.
+ * Returns the remote sourceInfo for a given source name.
+ *
+ * @param {string} sourceName - The source name.
+ * @returns {TPCSourceInfo}
+ */
+TraceablePeerConnection.prototype.getRemoteSourceInfoBySourceName = function(sourceName) {
+    return cloneDeep(this._remoteSsrcMap?.get(sourceName) ?? null);
```

Comment:
```suggestion
    return cloneDeep(this._remoteSsrcMap.get(sourceName));
```

Reply:
Now that the map always exists there is no need to optional-chain the call. Also, any reason to prefer a null result vs undefined? The map will return undefined if the key is not in there, which should suffice and simplifies the code a bit?